che-theia-java-extension/src/browser/navigation/find-implementers.ts

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which is available at http://www.eclipse.org/legal/epl-2.0.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import { inject, injectable } from "inversify";
+import { QuickOpenModel, QuickOpenItem, KeybindingRegistry, QuickOpenService, QuickOpenMode, KeybindingContribution } from "@theia/core/lib/browser";
+import { CommandRegistry, CommandContribution, CommandHandler, Command, MenuModelRegistry, MenuContribution } from "@theia/core";
+import { ILanguageClient, TextDocumentPositionParams, ExecuteCommandRequest, TextDocumentIdentifier, SymbolKind } from "@theia/languages/lib/browser";
+import { LanguageClientProvider } from "@theia/languages/lib/browser/language-client-provider";
+import { EditorManager, EDITOR_CONTEXT_MENU } from "@theia/editor/lib/browser";
+import { FIND_IMPLEMENTERS_COMMAND } from "../che-ls-jdt-commands";
+import URI from "@theia/core/lib/common/uri";
+import { Range } from 'vscode-languageserver-types';
+import { JavaKeybindingContexts } from "../java-keybinding-contexts";
```

Comment:
FYI in theia, there will have a rule to enforce quotes (here some imports are using single and double quotes)